subsys/bluetooth/services/cgms/cgms_socp.c

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <zephyr/types.h>
+#include <string.h>
+#include <zephyr/zephyr.h>
+
+#include <bluetooth/services/cgms.h>
+#include "cgms_internal.h"
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_DECLARE(cgms, CONFIG_BT_CGMS_LOG_LEVEL);
+
+/**@brief Specific Operation Control Point opcodes. */
+#define SOCP_OPCODE_RESERVED                          0x00
+#define SOCP_WRITE_CGM_COMMUNICATION_INTERVAL         0x01
+#define SOCP_READ_CGM_COMMUNICATION_INTERVAL          0x02
```

Comment:
```suggestion
#define CGMS_SOCP_OPCODE_RESERVED                          0x00
#define CGMS_SOCP_OPCODE_COMMUNICATION_INTERVAL_WRITE         0x01
#define CGMS_SOCP_OPCODE_COMMUNICATION_INTERVAL_READ          0x02
(....)
```

Please use an ```enum``` for this group defintion